apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClient.InsertData.cs

```diff
@@ -546,7 +546,7 @@ public abstract partial class BigQueryClient
         /// <returns>A data upload job.</returns>
         public virtual BigQueryJob UploadParquet(string datasetId, string tableId, Stream input, UploadParquetOptions options = null) =>
             UploadParquet(GetTableReference(datasetId, tableId), input, options);
-
+        
```

Comment:
Nit: added spaces at the beginning of the line, here and in several other lines on this file. It's the codegen I think.

Reply:
Thanks - will tweak until the difference looks better, then merge.